client/js/app.js

```diff
@@ -95,6 +97,15 @@ const init = () => {
     visibleClass: 'tabpanel--is-visible'
   }));
 
+  if (maps.length > 0) {
+    window.WC = {
```

Comment:
This will override the previous `WC` with the `featuresCohort` no?

Reply:
Why not stick this logic in the `map.js`?
It seems strangely separated and marooned here?